### PR TITLE
Ruby html-formatter

### PR DESCRIPTION
**Pull request by vincent-psarga**
Create `html-formatter` for Ruby, based on what has been done for Java and Javascript.

### CODE REVIEW

File: html-formatter/ruby/assets/.gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+*.html
+*.js
+*.css
```
Comment (mpkorstanje) [inline html-formatter/ruby/assets/.gitignore]: Are these removed again when `make clean` is used?

File: html-formatter/ruby/assets/index.mustache.html [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<title>Cucumber</title>
```
Comment (mpkorstanje) [inline html-formatter/ruby/assets/index.mustache.html]: This file is duplicated 3 times over now. Perhaps we should copy it from the `html-formatter` directory.

File: html-formatter/ruby/assets/.gitignore (hunk 1 above)
Comment (vincent-psarga) [reply html-formatter/ruby/assets/.gitignore]: now they should be, thanks for pointing that out :)

File: html-formatter/ruby/assets/index.mustache.html (hunk 2 above)
Comment (vincent-psarga) [reply html-formatter/ruby/assets/index.mustache.html]: It should have been removed with commit 14a6e12, but it did not. Normally that's ok now.

File: html-formatter/ruby/Makefile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+assets/cucumber-html.js:
+	mkdir -p $(@D)
+	cp ../javascript/dist/main.js $@
+
+assets/index.mustache.html:
+	mkdir -p $(@D)
+	cp ../javascript/dist/src/index.mustache.html $@
+
+
+clean:
```
Comment (mpkorstanje) [inline html-formatter/ruby/Makefile]: Does this override the `clean` directive in the included make file or does it extend it?

Comment (vincent-psarga) [reply html-formatter/ruby/Makefile]: it extends it:

```shell
╭─vincent.pretre@Vincents-MacBook-Pro ~/dev/open-source/cucumber/html-formatter/ruby ‹ruby-2.6.3› ‹html-formatter-for-ruby› 
╰─$ make clean
rm -f .deps .linked .tested* Gemfile.lock *.gem
rm -rf acceptance features/all_green.feature features/statuses.feature assets/*.css assets/*.js assets/*.html
```

First line comes from `default.mk`, second one from `Makefile`

Comment (mpkorstanje) [reply html-formatter/ruby/Makefile]: Cool. Still a bit hazy on how includes actually work.

**Pull request merged**